backend/llm/qa_base.py

```diff
@@ -103,7 +103,7 @@ def _create_llm(
         :param callbacks: Callbacks to be used for streaming
         :return: Language model instance
         """
-        return ChatOpenAI(
+        return ChatLiteLLM(
```

Comment:
Better do something like:
```python
if chatLiteLlm
  return ChatLiteLLM(
      temperature=temperature,
      model=model,
      streaming=streaming,
      verbose=False,
      callbacks=callbacks,
      openai_api_key=self.openai_api_key,
  )

return ChatOpenAI(
            temperature=temperature,
            model=model,
            streaming=streaming,
            verbose=False,
            callbacks=callbacks,
            openai_api_key=self.openai_api_key,
        )  # pyright: ignore reportPrivateUsage=none
```

This add new .env variable that will pick between ChatOpenAi and ChateLite instead of replacing it